### DISCUSSION TITLE
Cloud-init into Windows and Linux Image?

### DISCUSSION THREAD

**Discussion by hung6525** [vmware/packer-examples-for-vsphere#177] [Help] [answered]
Any plans on including Cloud-ini into Windows and Linux Image?

**Comment by tenthirtyam** [comment, marked as answer]
Yes!

https://github.com/vmware-samples/packer-examples-for-vsphere/discussions/158#discussioncomment-2231682

**Comment by tenthirtyam** [comment]
- #692 
- #691
